front/app/modules/commercial/insights/admin/containers/Insights/Details/Network/index.tsx

```diff
@@ -37,6 +37,19 @@ const chargeStrength = -25;
 const chargeDistanceMax = 80;
 const linkDistance = 40;
 
+const nodeColors = [
+  colors.clGreen,
+  colors.clBlue,
+  colors.clRed,
+  colors.adminOrangeIcons,
+  colors.adminTextColor,
+  colors.facebookMessenger,
+  colors.facebook,
+  colors.label,
+  '#0DA796',
+  '#934E6F',
```

Comment:
No because the colors in styleUtils are used all over the app while these are very particular and we introduce them for the viz only :) Would have been ideal to only use existing colors but was not possible to come up with 10 😅